### ISSUE TITLE
Not read TreeView nodes by JAWS and NVDA in Chrome

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-ui-core#3771]
### Bug report
Currently, TreeView nodes are not read by JAWS and NVDA upon navigation in Chrome.
**NOTE:** If the attribute **role="tree"** is applied to the _div_ wrapper, instead of the _ul_ element, the nodes are read. The _aria-activedescendant_ attribute is currently applied to the _div_ . Please note that upon testing with the role applied to the _div_, the nodes are read in IE and Chrome, but there are JavaScript errors in FF when navigating between nodes when JAWS is running, so this issue should be investigated and tested.

### Reproduction of the problem
1. Go to http://runner.telerik.io/fullscreen/IYijE
2. Focus the TreeView by pressing Alt + w
3. Move between nodes using Arrow keys

### Current behavior
TreeView nodes are not read.

### Expected/desired behavior
TreeView nodes are read, as in IE or Firefox

### Environment

* **Kendo UI version:** 2017.3.1026
* **Browser:** [Chrome 62.0.3202.94]

**Comment by veselints** [comment]
As per [the WCAG guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2a.html), the `role="tree"` attribute should be applied to the parent `<ul>` DOM element.
